github/repos_deployments_test.go

```diff
@@ -108,6 +108,28 @@ func TestRepositoriesService_ListDeploymentStatuses(t *testing.T) {
 	}
 }
 
+func TestRepositoriesService_GetDeploymentStatus(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/deployments/3/statuses/4", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeDeploymentStatusPreview)
+		fmt.Fprint(w, `{"id":4}`)
+	})
+
+	deploymentStatus, _, err := client.Repositories.GetDeploymentStatus("o", "r", 3, 4)
+	if err != nil {
+		t.Errorf("Repositories.GetDeploymentStatus returned error: %v", err)
+	}
+
+	want := &DeploymentStatus{ID: Int(4)}
+
```

Comment:
Super nitpicky, sorry, but mind removing this blank line? Because neither the test above nor below has one between `want :=` and `if !reflect.DeepEqual`, so it's inconsistent. It's okay either way.